modules/channel-messenger/src/backend/messenger.ts

```diff
@@ -28,13 +28,13 @@ export class MessengerService {
   async initialize() {
     const config = (await this.bp.config.getModuleConfig('channel-messenger')) as Config
 
-    if (!config.verifyToken || !config.verifyToken.length) {
+    if (!config.verifyToken?.length) {
       throw new Error(
         'You need to set a non-empty value for "verifyToken" in data/global/config/channel-messenger.json'
       )
     }
 
-    if (!config.appSecret || !config.appSecret.length) {
+    if (!config.appSecret?.length) {
```

Comment:
Since you are adding default values to appSecret and verifyToken, then these validations should also check if it's the default: `if(!config.appSecret?.length || config.appSecret === 'appsecret'`